[ONNX] Move large scale models without non-persistent buffers to runtime test

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #108084

Fixes https://github.com/pytorch/pytorch/issues/107715

Update models with their config to save CI running time and memories. Move some of models that doesn't need non-persistent buffers to runtime test.

titaiwangms committed:
[ONNX] Move large scale models without non-persistent buffers to runtime test

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move large scale models without non-persistent buffers to runtime test"


Fixes https://github.com/pytorch/pytorch/issues/107715

Update models with their config to save CI running time and memories. Move some of models that doesn't need non-persistent buffers to runtime test.

[ghstack-poisoned]

titaiwangms commented on test/onnx/test_fx_to_onnx_with_onnxruntime.py:
```diff
@@ -1047,6 +1047,119 @@ def create_kwargs():
+
+    @pytorch_test_common.xfail(
+        "Not decorated with xfail because CI doesn't have enough memory to run and then fail."
+        "AssertionError: whole graph export entails exactly one guard export"
```
Same as https://github.com/pytorch/pytorch/issues/102788

thiagocrepaldi commented on test/onnx/test_fx_to_onnx.py:
```diff
@@ -434,25 +434,6 @@ def test_fake_tensor_mode_huggingface_gpt2(self):
             onnx.checker.check_model(export_output.model_proto)
             onnx.shape_inference.infer_shapes(export_output.model_proto)
 
-    def test_fake_tensor_mode_huggingface_bigscience_bloom(self):
```
it seems this one was deleted instead of moved?

titaiwangms replied:
I already have bloom in runtime.

thiagocrepaldi commented on test/onnx/test_fx_to_onnx.py:
```diff
@@ -571,36 +510,6 @@ def test_fake_tensor_mode_huggingface_tiiuae_falcon(self):
-        "torch._dynamo.exc.UserError: Dynamic control flow is not supported at the moment. "
-        "Please use functorch.experimental.control_flow.cond to explicitly capture the control flow"
-    )
-    def test_fake_tensor_mode_huggingface_mosaicml_mpt_7b(self):
```
was it renamed to `test_fake_tensor_mode_huggingface_mosaicml_mpt` on purpose? any arch change?

titaiwangms replied:
Yes the name 7b is not true anymore, as it's built from config.

thiagocrepaldi commented:
Maybe mark this for merge on pytorch 2.1?

titaiwangms commented:
> Maybe mark this for merge on pytorch 2.1?

It's fine not user-facing.

thiagocrepaldi commented:
@pytorchbot merge

thiagocrepaldi commented:
good job

titaiwangms commented:
@pytorchbot merge

BowenBao commented on test/onnx/test_fx_to_onnx_with_onnxruntime.py:
```diff
@@ -1047,6 +1047,119 @@ def create_kwargs():
+            export_within_fake_mode=self.export_within_fake_mode,
+        )
+
+    @pytorch_test_common.xfail(
```
It is confusing that the reason says `Not decorated with xfail because ...`, but it is decorated with xfail.

titaiwangms replied:
Removed

titaiwangms committed:
Update on "[ONNX] Move large scale models without non-persistent buffers to runtime test"


Fixes https://github.com/pytorch/pytorch/issues/107715

Update models with their config to save CI running time and memories. Move some of models that doesn't need non-persistent buffers to runtime test.

[ghstack-poisoned]

titaiwangms committed:
Update on "[ONNX] Move large scale models without non-persistent buffers to runtime test"


Fixes https://github.com/pytorch/pytorch/issues/107715

Update models with their config to save CI running time and memories. Move some of models that doesn't need non-persistent buffers to runtime test.

[ghstack-poisoned]

titaiwangms commented:
@pytorchbot merge

titaiwangms commented:
@pytorchbot merge -f "unrelated flaky trunk"
